test/extensions/filters/http/cache/http_cache_utils_test.cc

```diff
@@ -75,6 +75,37 @@ TEST_P(EffectiveMaxAgeTest, EffectiveMaxAgeTest) {
             std::chrono::seconds(GetParam().effective_max_age_secs));
 }
 
+void testReadAndRemoveLeadingDigits(absl::string_view s, int64_t expected,
```

Comment:
Use `input` rather than `s` here -- `s` might be an appropriate loop index but we should use words for variable names with a larger scope.